website/pages/blog/announcing-nextra-2.mdx

```diff
@@ -0,0 +1,344 @@
+
+````md filename="Markdown"
+```jsx filename="My File" showLineNumbers /App/ {2}
+function App() {
+  return <div>Hello</div>
+}
+```
+````
+
+will be rendered as:
+
+```jsx filename="My File" showLineNumbers /App/ {2}
+function App() {
+  return <div>Hello</div>
+}
+```
+
+#### Next SEO Builtin
+
+Out of box `nextra-theme-docs` has Next SEO installed. By default, in front matter you can set `title`,
+`description`, `canonical` and `openGraph` and they will be passed directly to `<NextSeo />` component.
+
+```md filename="Markdown"
+---
+title: Nextra 2
+description: Nextra – Next.js Static Site Generator
+---
+```
+
+will be passed to Next SEO and rendered on page as:
+
+```html
+<head>
+  <title>Nextra 2</title>
+  <meta name="og:title" content="Nextra 2" />
+  <meta name="description" content="Nextra – Next.js Static Site Generator" />
+  <meta name="og:description" content="Nextra – Next.js Static Site Generator" />
+</head>
+```
+
+<Callout>
+  You can manually pass Next SEO props with `useNextSeoProps` theme option. [Read
+  more](https://nextra.site/docs/docs-theme/theme-configuration#seo-options) in the docs.
+</Callout>
+
+#### I18n Support
+
+`nextra-theme-docs` comes with support i18n in your website.
+
+To use it, provide Next.js' `i18n` field with `locales` and `defaultLocale` set up:
+
+```js filename="next.config.js"
+export default withNextra({
+  i18n: {
+    locales: ['en-US', 'fr-FR'],
+    defaultLocale: 'en-US'
+  }
+})
+```
+
+Add Nextra's middleware:
+
+```js filename="middleware.js"
+export { locales as middleware } from 'nextra/locales'
+```
+
+Create your `_meta` and page files with locales suffixes:
+
+```text
+├── pages
+│   ├── _meta.en-US.json
+│   ├── _meta.fr-FR.json
+│   ├── index.en-US.mdx
+│   ├── index.fr-FR.mdx
+```
+
+```mdx filename="index.en-US.mdx"
+# 🇺🇸 Hello Nextra 2
+```
+
+```mdx filename="index.fr-FR.mdx"
+# 🇫🇷 Bonjour Nextra 2
+```
+
+[Learn more](https://nextra.site/docs/guide/i18n) on docs.
+
+#### LTR/RTL Direction Support
+
+Your website can be fully mirrored with RTL direction. All what you need it provide `dir: 'rtl'` in
+your `theme.config.jsx` file.
+
+<NextImage
+  src={nextraRtl}
+  className="mt-6 rounded-3xl border border-gray-700"
+  alt="Example of Nextra page with RTL direction"
+  loading="lazy"
+/>
+
+#### A11y
+
+Accessibility is all, Nextra respects system preferences, animation will be reduced with `reduce-motion` mode,
+colors will be adjusted in `contrast-more` mode.
+
+<NextImage
+  src={nextraContrastMode}
+  className="mt-6 rounded-3xl border border-gray-700"
+  alt="Example of Nextra page with contrast mode enabled"
+  loading="lazy"
+/>
+
+## Conclusion
+
+[The Guild](/) discovered Nextra 2 from the first betas and actively participated in development and improvement this library.
+At the end we migrated all of our documentation websites, and received a bunch of positive feedback
+from our community ❤️.
```

Comment:
maybe you can also expand more on The Guild's docs and all the difference features we've added (like Giscus for example)